lib/child_process.js

```diff
@@ -63,7 +64,9 @@ exports.fork = function(modulePath /*, args, options*/) {
 
   if (pos < arguments.length && arguments[pos] != null) {
     if (typeof arguments[pos] !== 'object') {
-      throw new TypeError('Incorrect value of args option');
+      throw new errors.TypeError('ERR_INVALID_OPT_VALUE',
```

Comment:
I suppose `ERR_INVALID_ARG_VALUE` would be better here, with the `args` name (this is not an options object).